### DISCUSSION TITLE
这样的换行要修改什么

### DISCUSSION THREAD

**Discussion by Jeamxyz** [ppoffice/hexo-theme-icarus#1004] [Q&A]
如图所示，一个困扰我许久的问题
</br>
![](https://raw.githubusercontent.com/jeam-xyz/image/main/202201030114220.png)
![](https://raw.githubusercontent.com/jeam-xyz/image/main/202201030114456.png)

**Comment by ppoffice** [comment]
https://github.com/ppoffice/hexo-theme-icarus/blob/master/layout/common/article.jsx#L71
加个`<br>`
